tests/system.rs

```diff
@@ -3330,6 +3330,34 @@ fn closure_bracket_scope_nested_match_outside() {
 }
 
 #[test]
+fn issue_754() {
```

Comment:
Please make the test name descriptive of what it demonstrates. Adding a link to the GitHub issue in the test's doc comment would be great though 😃